docs/compile-xtrabackup.md

```diff
@@ -94,28 +91,33 @@ directory:
     $ cd build
     ```
 
-3. Run cmake or cmake3. In either case, the options you need to use are the
+3. If you use a regular build, run cmake or cmake3. In either case, the options you need to use are the
 same.
 
 !!! note
  
     You can build *Percona XtraBackup* with man pages but this requires
     `python-sphinx` package which isn’t available from that main repositories
     for every distribution. If you installed the `python-sphinx` package you
-    need to remove the `-DWITH_MAN_PAGES=OFF` from previous command.
+    need to remove the `-DWITH_MAN_PAGES=OFF` from the following command.
+
+```{.bash data-prompt="$"}
+$ cmake -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF ..
+```
+
+If you use [Pro builds](./pxb-pro.md), run the following command
 
 ```{.bash data-prompt="$"}
-$ cmake -DWITH_BOOST=PATH-TO-BOOST-LIBRARY -DDOWNLOAD_BOOST=ON \
--DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF -B ..
+$ cmake -DPROBUILD=1 -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF ..
 ```
 
 ### Parameter Information
 
 | **Parameter** | **Description** |
 |---------------|-----------------|
```

Comment:
The table order should match the order of options in the command. 
DPROBUILD
DBUILD_CONFIG

and so on. 
Otherwise, it seems random and just stuck in.

Reply:
updated